src/utils.ts

```diff
@@ -17,3 +17,11 @@ export function extractAxiosErrorMessage(err: AxiosError): string {
 
   return errMsg;
 }
+
+export function isHexString(string: string) {
+  if (typeof string !== 'string') {
+    return false;
+  }
+
+  return /^([0-9a-fA-F]{2})+$/.test(string);
```

Comment:
We can go with this regex from a pragmatic point of view, but we must change this to use some kind of reliable library, add a TODO above please

Reply:
done